trajopt_examples/config/basic_cartesian_plan.json

```diff
@@ -11,7 +11,9 @@
       "type" : "joint_vel",
       "params":
       {
-        "coeffs" : [5]
+        "joint_name" : "all_joints",
+        "coeffs" : [5],
+        "penalty_type" : "SQUARED"
```

Comment:
Just curious, but when would you use the other penalty types?

Reply:
This will be addressed in another PR where we integrate the tolerances into the joint costs